tcp-to-blob/deploy/Dockerfile-text-canary

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Microsoft Corporation. All rights reserved.
+# Software is licensed under the MIT License. See LICENSE in the project
+# root for license information.
+
+ARG IMAGE_NAME=mcr.microsoft.com/mirror/docker/library/alpine
+ARG IMAGE_TAG=3.15
+
+FROM $IMAGE_NAME:$IMAGE_TAG
+
+RUN apk add --update nodejs
+
+WORKDIR /app/
+
+COPY ./dist/bundle/tcp-to-blob-text-canary.js .
```

```suggestion
COPY ./dist/bundle/tcp-to-blob-text-canary.js .

```
GitHub complaining about missing line ending